omero/sysadmins/public.txt

```diff
@@ -0,0 +1,135 @@
+Public data in the repository
+=============================
+
+The OMERO.web framework allows raw data to be published using built-in tools
+or supplied through webservices to the external web resources. This feature is
```

Comment:
Remove 'the' from 'external web resources' since this otherwise reads as if we're talking about a particular thing.
Maybe 'resources' isn't quite the right word, since *we* are a 'resource' to external web pages. External web pages may or may not be a 'resource'?